Freshli.Web/Views/AnalysisRequests/Show.cshtml

```diff
@@ -2,32 +2,36 @@
 
 <div>
     <p>LibYear analysis for @Model.Request.Url<p/>
```

Comment:
I think it makes sense to move this into the `if` block on line 8. 

If you want to always display this, I'd move it below the status message.

Reply:
Moved in 764ff76.